scripts/verify-genesis-locally.bash

```
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Verification works only after v1.2.0-rc.1 version
+TAG="${1:-v1.2.0-rc.1}"
+
+CURRENT_REV="$(git rev-parse HEAD)"
+TAG_REV="$(git rev-list -n 1 "$TAG")"
+``
```

Shellcheck (0.11.0)

[warning] Remove backticks to avoid executing output (or use eval if intentional).

(SC2092)